packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts

```diff
@@ -196,7 +199,7 @@ export class OnDemandDispatcher {
                     onMergeCallbackFn,
                 );
 
-                this.logger.logWarn(`Updated website page scan run state for a failed run.`, {
+                this.logger.logWarn(`Updated website page scan run state for abandon run.`, {
```

Comment:
Might be useful to have this log message different from the one at line 165 to make reasoning about the traces easier.

Reply:
It is different:
this.logger.logWarn('Updated **page** scan run state for abandon run.'
this.logger.logWarn('Updated **website** page scan run state for abandon run.'